tusclient/request.py

```diff
@@ -20,6 +25,7 @@ class TusRequest(object):
     def __init__(self, uploader):
         self.handle = pycurl.Curl()
         self.response_headers = {}
+        self.storage = io.StringIO()
```

Comment:
also should we rename `self.storage` to `self.output` maybe? Or something else that is more explanatory so it's not misleading?